ext/storage_sources/gcp_store/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+project(gcp_store CXX)
+
+find_package(google_cloud_cpp_common CONFIG REQUIRED)
+
+find_package(google_cloud_cpp_rest_internal CONFIG REQUIRED)
+
+find_package(google_cloud_cpp_storage CONFIG REQUIRED)
+
+# Create an intermediate library to build the helper library.
+add_library(gcp_bucket_util STATIC gcp_connection.cpp)
+set_property(TARGET gcp_bucket_util PROPERTY POSITION_INDEPENDENT_CODE ON)
+
+target_include_directories(
+    gcp_bucket_util
+    PUBLIC
+        ${CMAKE_CURRENT_SOURCE_DIR}
+)
+
+target_compile_options(
+    gcp_bucket_util
+    PRIVATE ${COMPILER_DIAGNOSTIC_CXX_FLAGS}
+)
+
+target_link_libraries(
+    gcp_bucket_util
+    PUBLIC
+        google-cloud-cpp::storage
+        google-cloud-cpp::common
+        google-cloud-cpp::rest_internal
+)
+
+# Build the gcp extension as a library.
+add_library(wiredtiger_gcp_store MODULE gcp_storage_source.cpp)
+target_include_directories(
+    wiredtiger_gcp_store
+    PRIVATE
+        ${CMAKE_SOURCE_DIR}/src/include
+        ${CMAKE_BINARY_DIR}/include
+        ${CMAKE_BINARY_DIR}/config
+        ${CMAKE_CURRENT_SOURCE_DIR}
+)
+
+target_compile_options(
+    wiredtiger_gcp_store
+    PRIVATE ${COMPILER_DIAGNOSTIC_CXX_FLAGS}
+)
+
+target_link_libraries(
+    wiredtiger_gcp_store
+    PRIVATE
+        gcp_bucket_util
+)
+    
```

Comment:
We can remove the blank line at the end of the file.

Reply:
Hi Tammy it has been addressed in: [e7cd853](https://github.com/wiredtiger/wiredtiger/pull/8688/commits/e7cd853492f5b96a6798679c3db140b75ce568eb)